addon/mixin.js

```diff
@@ -6,7 +6,7 @@ import {
 
 var get        = Ember.get;
 var set        = Ember.set;
-var keys       = Ember.keys;
```

Comment:
We should likely do `Object.keys || Ember.keys` and `Object.create || Ember.create`although this will spew deprecations in IE8, it wont break in IE*